obspy/io/mseed/util.py

```diff
@@ -105,27 +105,40 @@ def get_start_and_end_time(file_or_file_object):
 @deprecated("'getTimingAndDataQuality' has been renamed to "
             "'get_timing_and_data_quality'. Use that instead.")
 def getTimingAndDataQuality(*args, **kwargs):
-    return get_timing_and_data_quality(*args, **kwargs)
+    return get_flags(activity_flags=False, io_flags=False, *args, **kwargs)
```

Comment:
`t_quality=False` per default, nee d to set it `True` here? Also, maybe rename to `timing_quality`?